### PR TITLE
build: Dist ostree-sepolicy-private.h

### PR DESCRIPTION
**Pull request by cgwalters**
Should fix the Travis builds which actually generate a legacy tarball via
Automake.

### CODE REVIEW

**Comment by jlebon** [comment]
Don't mind me, just investigating the CI flakes!
@rh-atomic-bot try

File: Makefile-libostree-defines.am [hunk 1]
```diff
@@ -31,6 +31,7 @@ libostree_public_headers = \
 	src/libostree/ostree-diff.h \
 	src/libostree/ostree-gpg-verify-result.h \
 	src/libostree/ostree-sepolicy.h \
+	src/libostree/ostree-sepolicy-private.h \
```
Comment (jlebon) [inline Makefile-libostree-defines.am]: But we don't want this in `/usr/include` though, right?

Comment (cgwalters) [inline Makefile-libostree-defines.am]: Ooh yes, right you are!  👍

**Comment by jlebon** [comment]
@rh-atomic-bot r+ 6f7f6a8

**Comment by cgwalters** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**